tests/RobotFramework/tedge_connect/tedge_connect_test.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation    Run connection test while being connected and check the positive response in stdout
+...              disconnect the device from cloud and check the negative message in stderr
+...              Run sudo tedge connect c8y and check 
+
+Library    SSHLibrary
+Library    CryptoLibrary    variable_decryption=True
+Library    Dialogs
+
+Suite Setup            Open Connection And Log In
+Suite Teardown         SSHLibrary.Close All Connections
+
+
+*** Variables ***
+${HOST}           192.168.99.110    #Insert the IP address if the default should not be used
+${USERNAME}       pi    #Insert the username if the default should not be used
+${PASSWORD}       thinedge    #Insert the password if the default should not be used
+
+*** Tasks ***
+tedge_connect_test_positive
+    Execute Command    sudo tedge connect c8y    #Connecting to Cumulocity IoT
+    ${stdout}=    Execute Command    sudo tedge connect c8y --test    #Testing the status of the connection
+    Should Contain    ${stdout}    Connection check to c8y cloud is successful.    #Expected message
+    Log    ${stdout}
+
+tedge_connect_test_negative
+    Execute Command    sudo tedge disconnect c8y    #Disonnecting from Cumulocity IoT
+    ${stdout}    ${stderr}=    Execute Command    sudo tedge connect c8y --test    return_stderr=True    #Testing the status of the connection
```

Comment:
It's a question. What ` ${stdout}    ${stderr}=` part means? We will record both `stdout` and `stderr`? I'm just curious because the next two lines don't use `${stdout}`.

Reply:
> It's a question. What ` ${stdout} ${stderr}=` part means? We will record both `stdout` and `stderr`? I'm just curious because the next two lines don't use `${stdout}`.


${stdout} is just name of a variable, helps me to remember where I hae used it, it can be anything